### PR TITLE
Update edge-subscriptions.md

### PR DESCRIPTION
**Pull request by SignorelliDenis**
Org headers will not be preserved if Edge Subscription is not in place. We rely on Direct Trust to advertise X-AnonymousTLS and promote org headers. Otherwise, messages will never be considered "AuthAs:Internal" from an EXO mailbox.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/architecture/edge-transport-servers/edge-subscriptions.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ manager: serdars
 
 Edge Subscriptions are used to populate the Active Directory Lightweight Directory Services (AD LDS) instance on the Edge Transport server with Active Directory data. Although creating an Edge Subscription is optional, subscribing an Edge Transport server to the Exchange organization provides a simpler management experience and enhances antispam features. You need to create an Edge Subscription if you plan to use recipient lookup or safelist aggregation, or if you plan to help secure SMTP communications with partner domains by using Mutual Transport Layer Security (MTLS).
 
+> [!NOTE]
+> The Edge Subscription is mandatory if Edge Transport should handle hybrid mail flow. Organization headers are only promoted between Edge Transport and Mailbox Servers through the Direct Trust Authentication (aka Mutual TLS) and Edge Subscription is required to achieve this authentication method.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/edge-transport-servers/edge-subscriptions.md:24]:
```suggestion
> The Edge Subscription is mandatory if Edge Transport should handle hybrid mail flow. Organization headers are only promoted between Edge Transport and Mailbox servers through the Direct Trust Authentication (aka Mutual TLS) and Edge Subscription is required to achieve this authentication method.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @SignorelliDenis, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**